### PR TITLE
Replace command line args in deployment doc

### PR DESCRIPTION
**Pull request by jkowall**
## Which problem is this PR solving?
Part of https://github.com/jaegertracing/documentation/issues/750

## Description of the changes
Adjusting command line and other components from v1 to v2.

## Checklist
- [X] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [X] I have signed all commits

### CODE REVIEW

File: content/docs/next-release-v2/deployment.md [hunk 1]
```diff
@@ -37,109 +36,55 @@ There are orchestration templates for running Jaeger with:
...
-```sh
-docker run \
-  --rm \
-  -p5775:5775/udp \
-  -p6831:6831/udp \
-  -p6832:6832/udp \
-  -p5778:5778/tcp \
-  jaegertracing/jaeger-agent:{{< currentVersion >}} \
-  --reporter.grpc.host-port=jaeger-collector.jaeger-infra.svc:14250
+  jaeger_storage:
```
Comment (yurishkuro) [inline content/docs/next-release-v2/deployment.md:51]: We have dedicated pages for each storage backend now, I would rather consolidate everything about that backend there, including config examples. This page could be just about different deployment roles, how they are different, the ports, the APIs exposed.

File: content/docs/next-release-v2/deployment.md [hunk 2]
```diff
@@ -151,46 +96,7 @@ At default settings **jaeger-collector** exposes the following ports:
...
-  -p 16685:16685 \
-  -p 16686:16686 \
-  -p 16687:16687 \
-  -e SPAN_STORAGE_TYPE=elasticsearch \
-  -e ES_SERVER_URLS=http://<ES_SERVER_IP>:<ES_SERVER_PORT> \
-  jaegertracing/jaeger-query:{{< currentVersion >}}
-```
-
+!!! NEED TO FIX 2 sections BELOW
 ### Clock Skew Adjustment
```
Comment (yurishkuro) [inline content/docs/next-release-v2/deployment.md:100]: need to move elsewhere, this is not a "deployment" section. Technically it belongs to query/UI role, so could move there.

File: content/docs/next-release-v2/deployment.md (hunk 1 above)
Comment (jkowall) [reply content/docs/next-release-v2/deployment.md:51]: I put the snippets here just to provide some guidance. If you want me to just point to the storage pages, I can do that. Please clarify.

File: content/docs/next-release-v2/deployment.md (hunk 2 above)
Comment (jkowall) [reply content/docs/next-release-v2/deployment.md:100]: Do you want me to put an example UI/query role here for the deployment instead? I can add the snippets.

File: content/docs/next-release-v2/deployment.md (hunk 1 above)
Comment (yurishkuro) [reply content/docs/next-release-v2/deployment.md:51]: Yes, I would put them in the storage pages and link from here. I don't think we should focus so much on the backends in this top page.

File: content/docs/next-release-v2/deployment.md (hunk 2 above)
Comment (yurishkuro) [reply content/docs/next-release-v2/deployment.md:100]: we need to mention the UI/query role on this page, but the actual configuration details can go to its own dedicated paged.

**Comment by jkowall** [comment]
Should be done with the updates and examples. This should fix : https://github.com/jaegertracing/documentation/issues/750

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**